schema/cassandra/temporal/schema.cql

```diff
@@ -25,6 +25,8 @@ CREATE TABLE executions (
   replication_encoding           text,
   timer                          blob,
   timer_encoding                 text,
+  visibility_task_data           blob,
+  visibility_task_encoding       text,
```

Comment:
i may prefer following the existing naming convention
```
  visibility_data           blob,
  visibility_encoding       text,
```

Reply:
This is what I started with, and it looks confusing. Together with `visibility_ts` which has nothing to do with visibility task. Yes, it would be great to rename other fields too but this is not an easy thing. But it is never too late to start doing things right. What do you think?

Reply:
> But it is never too late to start doing things right. What do you think?
If we can migrate existing column names, yes, if not, i would prefer to following the existing convention.